src/certificates/endpoints.ts

```diff
@@ -21,23 +21,18 @@ import {
     AccountAdminApi as AdminApi,
     DeveloperApi as AccountDeveloperApi
 } from "../_api/iam";
-import {
-    ServerCredentialsApi as ServerApi,
-    DeveloperCertificateApi as CertDeveloperApi
-} from "../_api/connector_ca";
+import { ExternalAPIApi as ConnectorApi } from "../_api/connector_ca";
```

Comment:
That surprise me. In Python SDK `ServerCredentialsApi` still exists. It's strange that now it is generated in JS as `ExternalAPIApi`

Reply:
There are 3 APIs in the JS SDK, ExternalAPIApi replicates all the others, so I switched to use just that one :)